src/NuGet.Clients/NuGet.PackageManagement.UI/Options/MappingUIDisplay.cs

```diff
@@ -0,0 +1,35 @@
+using System.Collections.Generic;
+using NuGet.VisualStudio.Internal.Contracts;
+
+namespace NuGet.Options
+{
+    public class MappingUIDisplay
+    {
+        public string ID { get; set; }
+        public List<PackageSourceContextInfo> Sources { get; private set; }
+
+        //View binds to this string
+        public string SourcesString
+        {
+            get
+            {
+                string sourcesString = "";
+                for (int i = 0; i < Sources.Count; i++)
+                {
+                    sourcesString += Sources[i].Name;
+                    if (i < Sources.Count - 1)
+                    {
+                        sourcesString += ", ";
+                    }
+                }
+                return sourcesString;
```

Comment:
I believe code line 16-25 can be replaced with `return string.Join(", ", Sources.Select(s => s.Name))`, it would be more readable.

Reply:
Thanks. Fixed that